src/Client/ConcordiumClient.cs

```diff
@@ -545,6 +549,112 @@ await Task.WhenAll(response.ResponseHeadersAsync, response.ResponseAsync)
             .ConfigureAwait(false);
     }
 
+    /// <summary>
+    /// Return a stream of blocks that arrive from the time the query is made onward.
+    /// This can be used to listen for incoming blocks.
+    /// </summary>
+    /// <param name="token">Cancellation token</param>
+    /// <returns>A stream of blocks that arrive from the time the query is made onward.</returns>
+    /// <exception cref="RpcException">
+    /// RPC error occurred, access <see cref="RpcException.StatusCode"/> for more information.
+    /// <see cref="StatusCode.Unimplemented"/> indicates that this endpoint is disabled in the node.
+    /// </exception>
+    public IAsyncEnumerable<ArrivedBlockInfo> GetBlocks(CancellationToken token = default)
+    {
+        var response = this.Raw.GetBlocks(token);
+        return response.ResponseStream.ReadAllAsync(token).Select(ArrivedBlockInfo.From);
+    }
+
+    /// <summary>
+    /// Return a stream of blocks that are finalized from the time the query is
+    /// made onward. This can be used to listen for newly finalized blocks. Note
```

Comment:
This note is it correct?

If you are using bounded channels it may be correct but `IAsyncEnumerable` doesn’t drop things. 

It is because it is dropped on the server side?

Reply:
If the client is too slow at consuming data then the server will skip the blocks. It has a short queue only.

Reply:
So this is fine? Should I just resolve this?

Reply:
Yes, this is fine